src/java/htsjdk/variant/variantcontext/CommonInfo.java

```diff
@@ -41,7 +42,9 @@
  *
  * @author depristo
  */
-public final class CommonInfo {
+public final class CommonInfo implements Serializable {
```

Comment:
what about the attributes in CommonInfo?

The declaration is `private Map<String, Object>`
serialization will break on this - Object can be anything.